datasets/cms_medicare/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import fnmatch
+import json
+import logging
+import os
+import pathlib
+import typing
+from zipfile import ZipFile
+
+import pandas as pd
+import requests
+from google.cloud import storage
+
+
+def main(
+    source_url: str,
+    source_file: pathlib.Path,
+    target_file: pathlib.Path,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+    headers: typing.List[str],
+    rename_mappings: dict,
+    pipeline_name: str,
+) -> None:
+
+    logging.info("Creating 'files' folder")
+    pathlib.Path("./files").mkdir(parents=True, exist_ok=True)
+
+    logging.info(f"Downloading file {source_url}")
+    download_file(source_url, source_file)
+
+    logging.info(f"Opening file {source_file}")
+
+    PIPELINES_NAME_INPATIENT = [
+        "inpatient_charges_2011",
+        "inpatient_charges_2012",
+        "inpatient_charges_2013",
+        "inpatient_charges_2014",
+        "inpatient_charges_2015",
+    ]
+    PIPELINES_NAME_OUTPATIENT = [
+        "outpatient_charges_2011",
+        "outpatient_charges_2012",
+        "outpatient_charges_2013",
+        "outpatient_charges_2014",
+    ]
```

Comment:
Please move these outside the `main` function. They're considered as global

Reply:
Done.